annotations/views.py

```diff
@@ -330,6 +332,59 @@ def collection_texts(request, collectionid):
     }
     return HttpResponse(template.render(context))
 
+class RecentActivity(models.Model):
+    def __init__(self):
+        self.appellation = 0
+        self.text = 0
+
+def recent_activity(request):
+    from datetime import datetime, timedelta
+    from django.utils import timezone
+    template = loader.get_template('annotations/recent_activity.html')
+    time_threshold = datetime.now() - timedelta(days=1)
+    text_list = list(Text.objects.filter(added__gt=time_threshold).order_by('-added')[:10])
```

Comment:
@erickpeirson : We can change that to 7 days to consider activity for the complete week.